### PR TITLE
#101 Fixed disappearance of the 2nd screen's dialogs after rotating

### PR DESCRIPTION
**Pull request by NikitaSukhanov**
- Dialogs made into fragments.

### CODE REVIEW

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/ResultActivity.java [hunk 1]
```diff
@@ -16,13 +12,32 @@
...
+
+    /**
+     * Tag for fragment done dialog.
+     */
+    private static final String DIALOG_DONE_TAG = "fragmentDoneDialog";
+
+    /**
+     * Tag for fragment home-return dialog.
+     */
+    private static final String DIALOG_HOME_RETURN_TAG = "fragmentHomeReturnDialog";
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/ResultActivity.java]: The length of this line is over than 80.
You can use your class `StringKeys` for storage these variables.

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/StringKeys.java [hunk 2]
```diff
@@ -9,4 +9,6 @@
     static final String TOTAL_SCALE = "Total scale";
     static final String CENTER_RELATIVE_X = "Relative center X";
     static final String CENTER_RELATIVE_Y = "Relative center Y";
+    static final String DIALOG_HOME_RETURN_TAG = "Fragment home-return dialog";
+    static final String DIALOG_DONE_TAG = "Fragment done dialog";
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/result/StringKeys.java:36]: Where are the comments?

**Review by MaximSmolskiy — APPROVED** [review]
Please add comments and it will be great!

File: app/src/main/java/com/mithridat/nonoconverter/ui/result/StringKeys.java (hunk 2 above)
Comment (NikitaSukhanov) [reply app/src/main/java/com/mithridat/nonoconverter/ui/result/StringKeys.java:36]: There were no comments for all other fields, I thought it's quite unnecessary.

**Review by maik-nack — APPROVED** [review]
NICE

**Pull request merged**